collector/indices.go

```diff
@@ -500,6 +500,18 @@ func NewIndices(logger log.Logger, client *http.Client, url *url.URL, shards boo
 				},
 				Labels: indexLabels,
 			},
+			{
+				Type: prometheus.CounterValue,
```

Comment:
are you sure this is a counter? i.e. the value can only increase. If so, please add a `_total` suffix to the metric name. If this is not a counter, please change the type to `prometheus.GaugeValue`

Reply:
Yeah, you are right. It's definitely gauge. Thank you. Fixed!